python/tvm/rpc/client.py

```diff
@@ -332,7 +330,7 @@ def text_summary(self):
         sorted_server = sorted(data["server_info"], key=lambda x: x["key"])
         for item in sorted_server:
             addr = item["addr"]
-            res += "%21s    " % ":".join(addr)
+            res += "%21s    " % ":".join(map(str, addr))
```

Comment:
This fixes a bug introduced by #10641 where `tvm.exec.query_rpc_tracker` fails where some element in `addr` is integer